src/View/Window.vala

```diff
@@ -757,8 +755,10 @@ public class Files.View.Window : Hdy.ApplicationWindow {
 
     private void add_window (GLib.File location = GLib.File.new_for_path (PF.UserUtils.get_real_user_home ()),
                              ViewMode mode = ViewMode.PREFERRED) {
-
-        marlin_app.create_window (location, real_mode (mode));
+        with (new Window (marlin_app)) {
```

Comment:
I think we need to bump the minimum Vala version to use this syntax?

Reply:
yeah, it was introduced in vala 0.50, will it be fine to bump it? i can use the normal syntax if it preferred.

Reply:
Yes, should be fine. Version 0.56 is supplied with Horus and we will soon be switching to Gtk4 anyway. We need not worry about compatibility with previous versions of elementary.

Reply:
I shall have to remember that syntax - looks neat.